### PR TITLE
Clear lintian

### PR DESCRIPTION
**Pull request by rogers0**
Includes the patch you gave me, and I add a few changes to clear lintian.
And set me as uploader, if you don't mind.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -2,6 +2,7 @@ Source: micro-evtd
 Section: utils
 Priority: optional
 Maintainer: Ryan Tandy <ryan@nardis.ca>
+Uploaders: Roger Shimizu <rogershimizu@gmail.com>
```
Comment (rtandy) [inline debian/control]: Yes, please: :+1:

File: debian/rules [hunk 2]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/make -f
 
+# Security Hardening
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
```
Comment (rtandy) [inline debian/rules]: Don't we get a lot of this for free just by building with newer dpkg-buildflags and possibly upgrading debhelper?

**Comment by rogers0** [comment]
> Please leave this one out

Sure. Just cherry-pick the commit you like.

> I don't have any objection to gbp-pq, but is this adding any particular value?

If you haven't tried gbp-pq, I definitely recommend you try it once.
It will simplify the work flow on patch management.

You may find the links below useful:
- https://honk.sigxcpu.org/piki/development/debian_packages_in_git/

File: debian/rules (hunk 2 above)
Comment (rogers0) [inline debian/rules]: Yes, good idea to upgrade to compat/debhelper 10. Actually all my packages already started to use debhelper 10.
And even there's debhelper 10 support in backports.

Some benefits/features are introduced here:
- https://lists.debian.org/debian-devel/2016/04/msg00018.html

**Comment by rtandy** [comment]
I squashed the two "gbp pq" commits together, omitted my commit about GIPO, and merged the rest. Thank you!

**Pull request closed without merging**